README.md

```diff
@@ -1,2 +1,65 @@
 # good-graylog2
-Good reporter for graylog2
+_Good Reporter For Graylog2_
+
+## Usage:
+
+```
+'use strict';
+
+var Hapi = require('hapi');
+var good = require('good');
+var goodConsole = require('good-console');
+var graylogReporter = require('good-graylog2');
+var config = require('config');
+
+var server = new Hapi.Server();
+
+server.connection({
+  port: 1234,
+  routes: {cors: true}
+});
+
+
+server.route([
+  {
+    method: 'GET',
+    path: '/healthcheck',
+    handler: function(request, reply) {
+      reply('OK');
+    }
+  }
+]);
+
+var hapiOptions = {
+  reporters: [
+    {
+      reporter: graylogReporter,
+      events: {
+        response: '*',
+        log: '*',
+        error: '*'
+      },
+      config: {
+        service: 'service-name',
+        host: 'graylog.example.com',
+        port: '1234',
+        container: config.container // optional
```

Comment:
Actually Nick, since this becomes the `hostname`, we should just call this option `hostname`. You can certainly put the container id there, but it's still a hostname option.